spec/broker_type_spec.rb

```diff
@@ -137,6 +137,8 @@ def with_brokers_in(brokers)
       end
 
       it "should raise if the install template is present but unreadable" do
+        pending("Skipping this test because process is running with root user privileges and root can read everything") if Process.uid == 0
```

Comment:
Can we break this line so the line length is more standard with the rest of the file? e.g.
```
  pending("Skipping this test because process is running with " +
          "root user privileges and root can read everything") if Process.uid == 0
```